server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@GuruprashanthKrishnakumar you shouldn't need to duplicate this file, you should be able to reference the same one used to build the char driver.